releases/changelog.md

```diff
@@ -0,0 +1,73 @@
+# Changelog
+
+In our repositories we keep `CHANGELOG.md` files describing changes between
+releases of the project. In general we follow [keep
+a changelog][keepachangelog] guidelines.
+
+**The changelog should be able to answer two questions:**
+
+- Is there a reason to upgrade to the new version?
+- What breaking changes were introduced? How the migration to the new
+  version looks like?
+
+**Those questions should be answered from the perspective of the user of the
+repository.** The user can be something else for different types of codebases:
+
+- For CLI program: a person or entity calling the binary.
+- For managed app: a person or entity installing the application.
+- For library: another codebase using the library.
+- For operator: a person or entity creating the CR consumed by this operator.
+
+## Examples of reasons to upgrade
```

Comment:
I do not see the value in these examples - what do you want to explain with these reasons to upgrade?

Reply:
That they answer the question:

> - Is there a reason to upgrade to the new version?

And therefore should be included in the changelog. <- Maybe that's sentence that's missing or this is entirely stupid to add that section?